src/analyzer.ts

```diff
@@ -198,15 +214,17 @@ export class BuildAnalyzer {
    * Return _dependenciesOutputStream, which will contain fully loaded file
    * objects for each dependency after analysis.
    */
-  get dependencies(): NodeJS.ReadableStream {
+  dependencies(): NodeJS.ReadableStream {
```

Comment:
I agree with that reasoning there, even though the breaking change and churn is unfortunate.